jujugui/static/gui/src/app/components/generic-input/_generic-input.scss

```diff
@@ -1,10 +1,20 @@
 .generic-input {
+    position: relative;
+
     &__label {
-        // Needs to be relative to appear above the input.
-        position: relative;
-        margin: 4px 0 -23px 10px;
-        padding-top: 5px;
-        font-size: 0.688em;
+        // Needs to be positioned to appear above the input.
+        color: $cool-grey;
+        left: 10px;
+        position: absolute;
+        top: 25px;
```

Comment:
This label also needs a `user-select: none` as I've selected the label multiple times. Or maybe the label should be behind the input? I'll leave that fix up to you.